pandas/io/excel/_odfreader.py

```diff
@@ -148,6 +150,9 @@ def _is_empty_row(self, row) -> bool:
     def _get_cell_value(self, cell, convert_float: bool) -> Scalar:
         from odf.namespaces import OFFICENS
 
+        if str(cell) == "#N/A":
```

Comment:
are there tests for NA handling?

Reply:
do these exist?

Reply:
Yes: test_readers.py::TestReaders::test_excel_cell_error_na